airbyte-server/src/main/java/io/airbyte/server/handlers/OperationsHandler.java

```diff
@@ -25,28 +25,32 @@
 import io.airbyte.config.persistence.ConfigRepository;
 import io.airbyte.server.converters.OperationsConverter;
 import io.airbyte.validation.json.JsonValidationException;
+import jakarta.inject.Inject;
+import jakarta.inject.Singleton;
 import java.io.IOException;
 import java.util.ArrayList;
 import java.util.HashSet;
 import java.util.List;
 import java.util.UUID;
 import java.util.function.Supplier;
 
+@Singleton
 public class OperationsHandler {
 
   private final ConfigRepository configRepository;
   private final Supplier<UUID> uuidGenerator;
 
+  @Inject
```

Comment:
Is the `@Inject` needed?

Reply:
I added it mostly for documentation purposes as there are two constructors on this class.  Technically [it isn't needed](https://docs.micronaut.io/latest/guide/#beans) as 
> Micronaut supports the following types of dependency injection:
> - Constructor injection (must be one public constructor or a single constructor annotated with @Inject)
and there is only a single public constructor in this class.